skia-bindings/build_support/skia.rs

```diff
@@ -284,6 +284,16 @@ impl FinalBuildConfiguration {
                 cflags.push(&sysroot_arg);
             }
 
+            let jpeg_sys_cflags: Vec<String>;
+            if cfg!(feature = "use-mozjpeg-sys") {
+                let paths = cargo::env_var("DEP_JPEG_INCLUDE").expect("mozjpeg-sys include path");
+                jpeg_sys_cflags = std::env::split_paths(&paths).map(|arg| {
+                    format!("-I{}", arg.display())
+                }).collect();
+                cflags.extend(jpeg_sys_cflags.iter().map(|x| -> &str { x.as_ref() }));
+                args.push(("skia_use_system_libjpeg_turbo", yes()));
+            }
+
```

Comment:
In this file can you also change:
```diff
-.args(&["-C", config.output_directory.to_str().unwrap()])
+.args(&[":skia", "-C", config.output_directory.to_str().unwrap()])
```
It avoids some issues when building `rust-skia` directly, but also should compile faster as it doesn't build un-needed targets.

Reply:
Thanks, done